spec/command_spec.rb

```diff
@@ -36,6 +36,11 @@
       expect(add_sub.call(command).parent).to eq(command)
     end
 
+    it "can tell whether it is a parent command or not" do
+      expect(command.root?).to be true
+      expect(command_with_parent.root?).to be false
+    end
```

Comment:
You should ideally be testing the issue at hand (which is in `Presenter`)

Reply:
I removed this test for the time being and will write a better one.

Reply:
Take your time. Don't forget to remove `Command#root?` if it no longer has a role..

Reply:
Added tests and removed `Command#root?`.